packages/wallets/src/components/WalletListCardActions/WalletListCardActions.tsx

```diff
@@ -94,7 +94,11 @@ const WalletListCardActions: React.FC<TProps> = ({ accountsActiveTabIndex }) =>
                             }}
                             size='lg'
                         />
-                        <Text size='sm' weight={button.text === localize('Deposit') ? 'bold' : 'normal'}>
+                        <Text
+                            align='center'
+                            size='2xs'
+                            weight={button.text === localize('Deposit') ? 'bold' : 'normal'}
```

Comment:
@aizad-deriv Dude, why not add a `weight` property to the button object  inside of the `getWalletHeaderButtons`? 🤔